nodejs/nodejs-guestbook/kubernetes-manifests/guestbook-backend.deployment.yaml

```diff
@@ -36,3 +36,10 @@ spec:
               key: password
         - name: MONGO_HOST
           value: "mongo-service"
+        readinessProbe:
+          httpGet:
+              port: 8080
+              path: /messages
+          failureThreshold: 3
+          timeoutSeconds: 2
+          successThreshold: 2
```

Comment:
Keep successThreshold:1, if it succeeds once, there's no need to wait another 10s to call the pod ready.